src/app/components/multiple-draft-generator/multiple-draft-generator.component.html

```diff
@@ -15,21 +49,47 @@ <h5>Languages without a Draft</h5>
     </label>
   </div>
   <br />
-  <ngb-alert type="info" [dismissible]="false" *ngIf="saving">
-    Saving...
+
+  <ngb-alert type="success" [dismissible]="false" *ngIf="sucessfulMessage">
+    {{ sucessfulMessage }}
+  </ngb-alert>
+  <ngb-alert type="info" [dismissible]="false" *ngIf="alertMessage">
+    {{ alertMessage }}
   </ngb-alert>
-  <ngb-alert type="danger" [dismissible]="false" *ngIf="errorMessage">
-    {{ errorMessage }}
+  <ngb-alert
+    type="danger"
+    [dismissible]="false"
+    *ngFor="let error of errorMessage"
+  >
+    {{ error }}
   </ngb-alert>
-  <button class="btn btn-danger" (click)="cancel()">Cancel</button>
-  <button class="btn btn-success" (click)="showConfirmAlert()">
-    Generate drafts
-  </button>
-  <ngb-alert type="danger" [dismissible]="false" *ngIf="confirmMessage">
-    {{ confirmMessage }}
-    <button (click)="generateDrafts()" class="btn btn-dark">Confirm</button>
-    <button (click)="confirmMessage = null" class="btn btn-secondary">
+
+  <ngb-alert type="info" [dismissible]="false" *ngIf="confirmMessage">
+    <p>{{ confirmMessage }}</p>
+    <div class="btn-toolbar justify-content-between">
+      <button (click)="confirmMessage = null" class="btn btn-danger">
+        Cancel
+      </button>
+      <button (click)="publishOrCreateDrafts()" class="btn btn-success">
+        Confirm
+      </button>
+    </div>
+  </ngb-alert>
+
+  <div class="modal-footer justify-content-between">
+    <button
+      class="btn btn-secondary"
+      (click)="cancel()"
+      [disabled]="disableButtons"
+    >
       Cancel
     </button>
-  </ngb-alert>
+    <button
+      class="btn btn-success"
+      (click)="showConfirmAlert()"
+      [disabled]="disableButtons"
+    >
+      {{ languageType === 'published' ? 'Generate drafts' : 'Publish drafts' }}
```

Comment:
I need to look at the actual UI, but I would guess we would just want it to say "Publish" instead of "Publish drafts".

Also `languageType` seems an odd name for the boolean setting

Reply:
We could even change it to "Publish selected", I'm down with whatever the God Tools team thinks makes more sense.

I could change `languageType` to `isADraft` or `isPublished`.

Reply:
"Publish selected" could work, I feel like "selected" would already be implied though

Reply:
for the name it could be something like:
- `generatorType`
- `actionType` with the types being `publish` or `createDrafts`

I view that option as selecting the type of action you are performing from the generate multiple UI

Reply:
I've updated the button name and renamed `languageType` to `actionType` with values of `publish` and `createDrafts`